.clang-format

```diff
@@ -1,3 +1,5 @@
+## I'm a little teapot, short and stout.
+
 ---
```

Comment:
ilya this code is awesome and elegant

---
**Sema Reaction:** :trophy: This code is awesome | **Sema Tags:** Elegant